sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
that's such an odd "feature" of go that I can't have a global variable that can be used across a package if the package is "main".  IMO there's no reason why all of the files for this cmd line couldn't be in one dir - its a pretty small package/exe.  oh well.